benchmarks/build.gradle

```diff
@@ -33,7 +33,7 @@ dependencies {
 }
 
 task qps_client(type: CreateStartScripts) {
-    mainClassName = "io.grpc.benchmarks.qps.QpsClient"
+    mainClassName = "io.grpc.benchmarks.qps.QpsAsyncClient"
```

Comment:
@ejona86 yes I agree, I think it even makes sense to put everything into 1 binary (server and client). As you said, it's easier to copy. I ll make it so in my next PR when I add the blocking client and server.